src/renderer/views/Channel/Channel.vue

```diff
@@ -369,6 +369,7 @@
         <ft-element-list
           v-if="!hideChannelCommunity && currentTab === 'community'"
           id="communityPanel"
```

Comment:
Still referenced by `aria-controls` somewhere

Reply:
We still have the ID. Just added it as a class as well

Reply:
Yup I know. It's for other reviewers (coz diff won't show `aria-controls`